filebeat/prospector/prospector.go

```diff
@@ -141,3 +144,7 @@ func (p *Prospector) stop() {
 		p.prospectorer.Stop()
 	}
 }
+
+func (p *Prospector) String() string {
```

Comment:
Is that a required method or just for convenience / logging?

Reply:
It's for logging purposes only, autodiscover will tell about new spawned prospectors/modules in the log output